metabase/metabase
# Fix x-axis default value in visualization settings

Closes https://github.com/metabase/metabase/issues/32179

### Description

Fixes x-axis default value in the visualization settings

### How to verify

1. New question -> Orders by Created At
2. Open the viz settings sidebar and go to "Axes" tab
3. Ensure X-Axis -> Label is not empty and shows correct value

### Checklist

- [x] X-axis label auto-populates
